docs/updating-spot-retries.md

```diff
@@ -0,0 +1,93 @@
+
+Since the default for spot retries is now `0`, you must actively enable a retry strategy if you want Nextflow to handle reclaimed spot instances automatically.
+
+## Possible actions
+
+There are four possible actions.
+
+### Do nothing
+
+If you do not configure anything, you will observe more pipeline failures when spot instances are reclaimed. This approach provides clearer visibility into failures. Failed tasks can be re-run with the `-resume` option. However, frequent task reclamation may lead to a higher failure rate and each retry requires manual intervention.
+
+:::{note}
+If you resume the pipeline using the resume option, it will pick up at the point the pipeline was interrupted and start with a retry of that task.
+:::
+
+### Re-enable spot retries
+
+You can re-enable spot retries at the provider level in your Nextflow configuration:
+
+```
+// nextflow.config
+aws {
+    batch {
+        maxSpotAttempts = 5
+    }
+}
+
+google {
+    batch {
+        maxSpotAttempts = 5
+    }
+}
+```
+
+The above example sets the maximum number of spot retries to `5` for both AWS and Google.
+
+### Make spot failures visible but retry them
+
+You can set `maxRetries` to enable Nextflow-level retries for any failure:
+
+```
+// nextflow.config
+process {
+    maxRetries = 5
+}
+```
+
+The above example sets retries to `5` for any failures, not just failures at the provider level.
+
+### Use Fusion snapshots
+
+If you have long-running tasks where losing progress due to spot reclamation is costly, consider [Fusion snapshots](https://docs.seqera.io/fusion/guide/snapshots) (if supported by your environment). Fusion snapshots allows you to resume a partially completed task on a new machine if a spot instance is reclaimed and reduce wasted compute time.
+
+Key features of Fusion snapshots:
+
+- Jobs do not need to start from scratch after reclamation.
+- Especially useful for tasks that take many hours or days to complete.
+- May significantly reduce costs and run times in high-reclamation environments.
+
+See [Fusion Snapshots for AWS Batch](https://docs.seqera.io/fusion/guide/snapshots) for more information.
+
+## Best practices
+
+Best practices for spot instance failures and retries:
+
+- **Evaluate job duration**: If your tasks are very long (for example, multi-hour, multi-day), spot instances can cause repeated interruptions. Consider using on-demand instances or Fusion snapshots.
+- **Set sensible retry limits**: If you enable spot retries, choose a retry count that balances the cost savings of spot usage against the overhead of restarting tasks.
+- **Monitor logs and exit codes**: Failures due to spot reclamation will now appear in Nextflow logs. Monitoring failures and fine-tune your strategy.
+- **Consider partial usage of spot**: Some workflows may mix on-demand instances for critical or long tasks and spot instances for shorter less critical tasks. This can optimize cost while minimizing wasted compute time.
```

Comment:
```suggestion
- **Consider partial usage of spot**: Some workflows may mix on-demand instances for critical or long tasks and spot instances for shorter, less critical tasks. This can optimize cost while minimizing wasted compute time.
```